File: exercicios/Python/1048/1048.py
valor = float(input())

if(valor <= 400):
    reaj = 0.15
    sal_reaj = valor + (valor * reaj)
    reaj_ganho = sal_reaj - valor
    p = 15
    print('Novo salario: {:.2F} '.format(sal_reaj))
    print('Reajuste ganho: {:.2F} '.format(reaj_ganho))
    print('Em percentual: {} %'.format(p))

elif(valor >= 400.01) and (valor <= 800):
    reaj = 0.12
    sal_reaj = valor + (valor * reaj)
    reaj_ganho = sal_reaj - valor
    p = 12
    print('Novo salario: {:.2F} '.format(sal_reaj))
    print('Reajuste ganho: {:.2F} '.format(reaj_ganho))
    print('Em percentual: {} %'.format(p))

elif(valor >= 800.01) and (valor <= 1200):
    reaj = 0.10
    sal_reaj = valor + (valor * reaj)
    reaj_ganho = sal_reaj - valor
    p = 10
    print('Novo salario: {:.2F} '.format(sal_reaj))
    print('Reajuste ganho: {:.2F} '.format(reaj_ganho))
    print('Em percentual: {} %'.format(p))

elif(valor >= 1200.01) and (valor <= 2000):
    reaj = 0.07
    sal_reaj = valor + (valor * reaj)
    reaj_ganho = sal_reaj - valor
    p = 7
    print('Novo salario: {:.2F} '.format(sal_reaj))
    print('Reajuste ganho: {:.2F} '.format(reaj_ganho))
    print('Em percentual: {} %'.format(p))

elif(valor >= 2000):
    reaj = 0.04
    sal_reaj = valor + (valor * reaj)
    reaj_ganho = sal_reaj - valor
    p = 4
    print('Novo salario: {:.2F} '.format(sal_reaj))
    print('Reajuste ganho: {:.2F} '.format(reaj_ganho))
    print('Em percentual: {} %'.format(p))

File: exercicios/JavaScript/1048/1048.js
const { parse } = require('path/posix');

var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var valor = parseFloat(lines.shift());

var n_Salario = parseFloat(0);
var reajuste = parseFloat(0);

if(valor > 0 && valor <= 400){
    reajuste = valor * 0.15;
    n_Salario = reajuste + valor;
    console.log("Novo salario: " + n_Salario.toFixed(2));
    console.log("Reajuste ganho: " + reajuste.toFixed(2));
    console.log("Em percentual: 15 %")
}else if(valor > 400 && valor <= 800){
    reajuste = valor * 0.12;
    n_Salario = reajuste + valor;
    console.log("Novo salario: " + n_Salario.toFixed(2));
    console.log("Reajuste ganho: " + reajuste.toFixed(2));
    console.log("Em percentual: 12 %")
}else if(valor > 800 && valor <= 1200){
    reajuste = valor * 0.10;
    n_Salario = reajuste + valor;
    console.log("Novo salario: " + n_Salario.toFixed(2));
    console.log("Reajuste ganho: " + reajuste.toFixed(2));
    console.log("Em percentual: 10 %")
}else if(valor > 1200 && valor <= 2000){
    reajuste = valor * 0.07;
    n_Salario = reajuste + valor;
    console.log("Novo salario: " + n_Salario.toFixed(2));
    console.log("Reajuste ganho: " + reajuste.toFixed(2));
    console.log("Em percentual: 7 %")
}else if(valor > 2000){
    reajuste = valor * 0.04;
    n_Salario = reajuste + valor;
    console.log("Novo salario: " + n_Salario.toFixed(2));
    console.log("Reajuste ganho: " + reajuste.toFixed(2));
    console.log("Em percentual: 4 %")
}